core/modules/filter/filter.pages.inc

```diff
@@ -628,6 +628,11 @@ function _filter_format_editor_link_url_validate(&$element, &$form_state) {
  */
 function _filter_get_file_id($href) {
   $result = 0;
+  
+  // Check URI for valid file scheme.
+  if(!file_valid_uri($href)){
```

Comment:
```suggestion
  if (!file_valid_uri($href)){
```
Just a missing space here.

Reply:
Oh yes, thanks! Can I just hit „Commit suggestion“ in such a case? I forgot. And the term „commit“ is irritating me.

Reply:
Yep that works. Thanks!